Dockerfile

```diff
@@ -10,5 +10,4 @@ COPY ./ ./
 VOLUME [ "/training-material" ]
 EXPOSE 8080
 
-ENTRYPOINT [ "npm",  "start", "--silent", "--" ]
-CMD ["--host", "0.0.0.0", "--env.material=/training-material"]
```

Comment:
Docker image not working without `--env.material=/training-material` args as default value is `training-material` (!= `/training-material`)

Reply:
Good catch!